testData/deviceReporter/installedSolutions.json

```diff
@@ -1,5 +1,9 @@
 [
     {
+        "id": "com.freedomscientific.jaws"
```

Comment:
This can be removed again. We made a decision a little while back that the installed Solutions file should only contain things that will be installed by default on the device (i.e. built-in features) - since we risk the system crashes if we attempt to run Jaws without it being installed.. If one wants to do development with jaws, one can always add it to their own installedSolutions file .

On non-development machines, we're using the dynamic device reporter (which uses the "isInstalled" block to decide whether it's installed) - so for these, the installedSolutions file doesn't matter